### ISSUE TITLE
Typography, colors, and layout  polishes

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/topcoder-app#1101]
There are some things to fix on the listing.

- [ ] wrong use of "Roboto-Bold" font instead of "Roboto" and font-weight — tags
- [ ] wrong sizing of tags; they must be 20px tall;
Desired CSS:
![screen shot 14](https://cloud.githubusercontent.com/assets/8484061/24424379/391611fc-13b5-11e7-8413-0e1bda6ee43f.png)

- [ ] Change the padding between prize and status columns to 20px; currently it is too big;
- [ ] Change the prize label from "total prize" to "purse"; add `text-wrap: no-wrap` so content will  never break into 2 lines; 
![screen shot 12](https://cloud.githubusercontent.com/assets/8484061/24424748/95920386-13b6-11e7-83c2-77a4278ffc95.png)

- [ ] User correct colors - on several places I saw `#999` instead of the correct gray;
- [ ] Font-weights reduction:
  - Current phase (medium → normal);
  - Track code (bold → medium);
  - Time left (medium → normal);Change color to $tc-gray-50
  - Number of users/submissions (medium → normal);
- [ ] Track progress in stalled/paused/delayed: from red to green; change the time remaining label to display the error. Too much red will create negative impact on the viewers
- [ ] Try to investigate if Chrome is rendering text with `geometricPrecision`, as the fonts doesn't look as they should. We might have a big problem with font loading from AWS, so please have someone look on what's the best way to serve our own web fonts from AWS. This one is pretty important as lack of rendering of fonts would result in bad UI.

**Screenshots:**

**Expected**
![screen shot 15](https://cloud.githubusercontent.com/assets/8484061/24424717/7f169496-13b6-11e7-95da-17f25ef24e68.png)

**Current**
![screen shot 16](https://cloud.githubusercontent.com/assets/8484061/24424727/86a0bd5e-13b6-11e7-9e6e-0506c26e7645.png)

**Comment by ajefts** [comment]
@ThomasKranitsas Another one if you want to grab it

**Comment by ThomasKranitsas** [comment]
@ajefts I don't have much time for this one. Sorry :)

**Comment by tladendo** [comment]
I'll get someone on this

**Comment by tladendo** [comment]
This is in progress and nearly done, just need to resolve some conflicts

**Comment by tladendo** [comment]
@vic-topcoder I've deployed what's been submitted, can you take a look in dev?

**Comment by vic-topcoder** [comment]
Yes, looks good so far. When can we expect the rest of the fixes?
